packages/scoutgame/src/builderNfts/accounting/getTransferSingleEvents.ts

```diff
@@ -48,11 +49,15 @@ export function getTransferSingleEvents({
     );
 }
 
-export function getStarterPackTransferSingleEvents({ fromBlock, toBlock }: BlockRange): Promise<TransferSingleEvent[]> {
+export function getStarterPackTransferSingleEvents({
+  fromBlock,
+  toBlock,
+  season
+}: BlockRange & { season?: ISOWeek }): Promise<TransferSingleEvent[]> {
```

Comment:
suggestion: make season required on low-level methods like these. The caller should always know which contract it's asking about, including which season